clients/privacy-center/cypress/e2e/show-modal.cy.ts

```diff
@@ -0,0 +1,53 @@
+import { stubConfig } from "../support/stubs";
+
+describe("Fides.showModal", () => {
+  describe("Overlay enabled", () => {
+    beforeEach(() => {
+      stubConfig({
+        options: {
+          isOverlayEnabled: true,
+        },
+        experience: {
+          show_banner: false,
+        },
+      });
+    });
+
+    it("Should add 'fides-overlay-modal-link-shown' class to body", () => {
+      cy.get("body").should("have.class", "fides-overlay-modal-link-shown");
+    });
+
+    it("Should allow showModal", () => {
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(100);
```

Comment:
Give enough time for the `Overlay.tsx` component to run the `useEffect` that sets `Fides.showModal`